udfs/tests/udf_test_utils.py

```diff
@@ -25,7 +25,6 @@
 
 from google.cloud import bigquery
 
-
```

Comment:
Please remove this change. Will keep PR focused on addition of UDF

Reply:
changes still pending

Reply:
@danieldeleo can you help me out with how to make this fix? I don't see where I made a change? Or perhaps you can suggest the git command to remove this frome the commit?

Reply:
This is the log output.

```
jrossthomson@drjay:~/bigquery-utils/udfs/tests$ git log -- test_run_udfs.py
commit 1e0bd0d42779114d1f5e475cd91d43d2fc049a62
Author: Daniel De Leo <danieldeleo@users.noreply.github.com>
Date:   Wed Jan 20 10:41:54 2021 -0500

    Adding CI/CD process for hosting minified javascript libraries (#187)

commit 1b6fc90eb0a434e36fdc7957fe030b7e622ddb6c
Author: Daniel De Leo <danieldeleo@users.noreply.github.com>
Date:   Tue Apr 7 10:27:43 2020 -0400

    Adding UDF Testing Framework (#36)
    
    * Adding UDF Unit Testing Framework
    
    Co-authored-by: tslr <tslr@google.com>
[hi on] jrossthomson@drjay:~/bigquery-utils/udfs/tests$ 
```